packages/plugins/rubygem-smart_proxy_dns_powerdns/rubygem-smart_proxy_dns_powerdns.spec

```diff
@@ -79,6 +77,9 @@ cp -pa .%{gem_instdir}/config/dns_powerdns.yml %{buildroot}%{foreman_proxy_setti
 %doc %{gem_instdir}/README.md
 
 %changelog
+* Wed Nov 13 2019 Tomer Brisker <tbrisker@gmail.com> - 0.4.0-2
```

Comment:
I'll release 0.5.0 with https://github.com/theforeman/smart_proxy_dns_powerdns/commit/deec2285488109c66b78c9ebae62456098602ee7 since that's cleaner IMHO.

Reply:
ping @ekohl - still waiting on the new release